cognite/neat/rules/models/information/_rules.py

```diff
@@ -171,15 +205,13 @@ class InformationProperty(SheetRow):
     comment: str | None = Field(alias="Comment", default=None)
     inherited: bool = Field(
         default=False,
+        exclude=True,
```

Comment:
This fixes the bug found by Kristian NEAT-417

Reply:
This is to exclude when exporting rules